### DISCUSSION TITLE
ForScan Connection not working

### DISCUSSION THREAD

**Discussion by markus2801** [EtoTen/stm32_can_sim#7] [General]
Hey, thanks for your work mate.
Everything works fine, except that ForScan doesn't find the APIM. 
Of course I did it with an "real" profile as described, but no matter what CAN pins on the APIM i use, the connection doesnt work.
Anyone having the same issue?

APIM is from 2018

**Comment by svenhe88** [comment]
Hi, 
same problem.... tried different can bus, PINs, but ForScan always says "No connection to car"

Also APIM from 2018 (2 different ones) 

Do I need a BCM??

regards

**Comment by EtoTen** [comment]
I am attaching a profile I got from forscan support. Here is the original description for it: "Please find the profile attached. Please use it with last official beta v2.4.3 (https://forscan.org/forum/viewtopic.php?f=13&t=19552). You need to unzip and import the attached profile in the Vehicle->Profiles section. Please note the profile is fake so use it only for the APIM bench programming.
"
[Ford Focus EcoBoost Gasoline Turbocharged Direct Injection 2_3L 2017 ( 2017 MY ) VIN WF0_________03158.profile.zip](https://github.com/EtoTen/stm32_can_sim/files/14633400/Ford.Focus.EcoBoost.Gasoline.Turbocharged.Direct.Injection.2_3L.2017.2017.MY.VIN.WF0_________03158.profile.zip)

It is very touchy, and I personally had issues before getting that profile. It could be easier to get a used BCM, which would trick ForScan into thinking you are scanning a real car.

**Comment by svenhe88** [comment]
Awesome,

thanks for the quick nice support.
This is the game changer, works fine with loaded profile, no need for bcm.


BR & thanks!

**Comment by svenhe88** [comment]
Hi,
do someone also have a profile from a car with sync2?
Is it also possible to program sync1 / sync2 with forScan on bench?
Changing VIN from sync2?
BR

**Comment by markus2801** [comment]
Hello,
I tried the new profile, but I had to use the latest ForScan beta (2.4.11), because I didnt find a link for 2.4.3.
However, it still doesnt work for me. I inspected the CAN with an Oscilloscope, and it seems like there is an 120 Ohms resistor missing.
Anyone having a scope for confirmation?
Or any other tips?

Regards

**Reply by EtoTen** [reply]
@markus2801 
Hmm, there is no harm in trying to add a resistor. 
The modified profile worked for me when I was messing with bench Forscan programming. Does your scope decode CAN? What are you seeing on it that makes you think a resistor missing?

That quote from Forscan, was from and email that was like 2 years ago, so 2.4.3 is long gone now. The profile should work with 2.4.11. Once you install the profile, does it give you the option to using an existing recorded profile once you try to connect? What step does it get stuck on? 

@svenhe88 did you use 2.4.11? Did it work for you or did you need a different version?

**Reply by svenhe88** [reply]
I used v2.3.5.9
You have to add a 120 Ohm resistor.
Open ForScan hit connect, an alert comes up "no vehicle found" use existing profile (import before)
Click on sync module, configure like you want.

BR

**Reply by markus2801** [reply]
Hello again,
I gave it a new try today, same setup as before and it now worked.
Setup: ForScan v2.4.11 beta, vLinker OBD Dongle, and 120 Ohms between CAN High and CAN Low.
Thanks for your help!
